.github/workflows/ci-build.yaml

```diff
@@ -149,10 +149,12 @@ jobs:
       - name: Install manifests
         run: make install PROFILE=${{matrix.profile}} STATIC_FILES=false
       - name: Build controller
-        run: make controller kit STATIC_FILES=false
+        run: make controller task STATIC_FILES=false
       - name: Build CLI
         run: make cli STATIC_FILES=false
         if: ${{matrix.test == 'test-api' || matrix.test == 'test-cli' || matrix.test == 'test-java-sdk' || matrix.test == 'test-python-sdk'}}
+      - name: Start port forward
+        run: ./hack/port-forward.sh
```

Comment:
this was removed in the `goreman` -> `kit` migration in https://github.com/argoproj/argo-workflows/pull/10599/files#diff-b8790dc873e80c2f7f63bf198b4da0147787ad8954767b839967d29f65155e2cL148, added it back now since `task` does not handle port forwarding

Reply:
this fixed the Controller tests. some Server tests are failing because apparently the Server didn't start at all. apparently `task` was waiting for `deps` to finish (which totally makes sense), but starting a servers will _never_ finish. so the `cmd`s need to be backgrounded properly. then everything works ok.

I think `kit`'s `ports` keyword (mentioned [above](https://github.com/argoproj/argo-workflows/pull/11928#discussion_r1343065238)) was just watching for ports to be opened before proceeding to the next task. nifty feature for servers.